_pkgdown.yml

```diff
@@ -36,6 +36,11 @@ navbar:
       href: news/index.html
 
 reference:
+- title: AE summary
+  contents:
+    - prepare_ae_summary
+    - format_ae_summary
+    - tlf_ae_summary
 - title: AE specification
   contents:
     - prepare_ae_specific
```

Comment:
Does `prepare_ae_exp_adj` belongs to the AE specification analysis? or it will require its own `format_` and `tlf_` functions?